### DISCUSSION TITLE
Ds718+ and ds923+ working fine !

### DISCUSSION THREAD

**Discussion by darkeness797** [bb-qq/r8152#348] [Show and tell]
Good morning,
A quick message to inform you that my ds718+ and ds923+ have been working in 2.5 Gb for 4 days now using the latest package available from "bb-qq", Club 3D usb/rj45 adapters and a 2.5 Gb Qnap switch
Many thanks for this project!

**Comment by HandiPC** [comment]
@darkeness797 , Hello, with which version of .spk did you do this for the DS923+ ?, Thank you for your response Darkeness797... and have a nice day.

**Reply by darkeness797** [reply]
Hello. I took braswell,r1000 and apollolake package but I don’t remember which one was ok for 923+ 
When installing package syno will tell you if it’s the good one for you :)
Latest version available 2.17.1
Keep me in touch if you havé troubles

**Reply by HandiPC** [reply]
Hi, on ds923+ "7.2.1-69057" with "r8152-r1000-2.17.1-1_7.2.spk" and "Club 3D usb/rj45 adapters and a 2.5 Gb", i received message "Failed to repair package [RTL8152/RTL8153 driver] ". Disappointed with this, hope for an update. Thank's for your help for this.

**Reply by darkeness797** [reply]
Have you tried with other packages? Same error message ?

**Reply by HandiPC** [reply]
Hi, I started again by entering the ssh command: "sudo install -m 4755 -o root -D /var/packages/r8152/target/r8152/spk_su /opt/sbin/spk_su" and then reinstalling the driver and everything works correctly?
Thank you for your help. 'DS923+ with r8152-r1000-2.17.1-1_7.2.spk. have a nice day.

**Comment by DonGeilo33** [comment]
R1000 worked for me as well on my 923+

**Comment by dnedbaylo** [comment]
It seems R1000 is correct version for Ryzen R1600 since https://www.amd.com/en/products/embedded/ryzen/ryzen-r1000-series.html
